### ISSUE TITLE
Files not formatted correctly upon creation

### ISSUE THREAD

**Issue by shimizurei** [Flameish/Novel-Grabber#22]
I tried to open this using Firefox's EPubReader extension, but was met with this message: `Unfortunately an unexpected error occurred opening the ebook. Please send the following error message to epubread@gmail.com. Thanks!: loadPage: TypeError: f.style is undefined`

Then I take it into Sigil to edit it and I'm met with this message: `This EPUB has HTML files that are not well formed. Sigil can attempt to automatically fix these files, although this can result in minor data loss. Do you want to automatically fix the files?`

I'm not sure what was changed, but I allowed Sigil to fix it.

**Comment by Flameish** [comment]
I took a look at EpubReader/Sigil and the reader is unable to open the file because certain html tags were not closed / formatted correctly. I fixed it for the parts I insert into the EPUB myself but I have no way of fixing it for the actual fetched chapter (the whole HTML code is being fetched directly) currently.

I will look into normalizing / fixing broken HTML tags but for now I can only suggest using a different EPUB reader. Calibre Ebook Viewer on PC and Moon+ Reader on mobile is what I'm using personally.
